### PR TITLE
Integrating ember-cli with rails. I migrated source over from another…

### PR DESCRIPTION
**Pull request by allthesignals**
… repo. Implements CORS changes.

### CODE REVIEW

**Comment by allthesignals** [comment]
I use an ES6 compiler so we'll have to ignore the ES6 complaints.

File: searchapp/app/components/mapbox-map.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    // recalculate bounds
+    this.recalculateBounds();
+
+  }.observes("developments"),
+
+  didInsertElement: function() {
+
+    // access token
+    mapboxgl.accessToken = 'pk.eyJ1Ijoid21hdHRnYXJkbmVyIiwiYSI6Ii1icTRNT3MifQ.WnayxAOEoXX-jWsNmHUhyg';
```
Comment (beechnut) [inline searchapp/app/components/mapbox-map.js]: :warning: Mapbox token is hardcoded.

**Comment by allthesignals** [comment]
not quite certain what's going on with this but I need to start cleaning up some local branches so let me know if you need me to merge this branch via cli myself or otherwise.

**Comment by beechnut** [comment]
I think you added a whole new .jshintrc, but I was thinking if you added the esnext option to searchapp/.jshintrc, a lot of these warnings might go away.

Also, I'm noticing that the Mapbox API token is still hardcoded -- can you address that?

File: searchapp/app/components/mapbox-map.js (hunk 1 above)
Comment (beechnut) [inline searchapp/app/components/mapbox-map.js]: It's a public API key, not private! Yay!

**Pull request merged**